sensor_msgs/NavSatFix[] waypoints
float64[] headings
---
bool success
